test/jdk/sun/security/pkcs11/Cipher/TestSymmCiphers.java

```diff
@@ -80,10 +80,11 @@ private static class CI { // class for holding Cipher Information
         new CI("DESede", "DESede", 408),
         new CI("AES", "AES", 128),
 
-        new CI("AES/CTR/NoPadding", "AES", 3200)
+        new CI("AES/CTR/NoPadding", "AES", 3200),
+        new CI("AES/CTS/NoPadding", "AES", 3200),
```

Comment:
Add more data sizes, e.g. not multiples of block sizes?

Reply:
These edge cases are covered by the new `TestCipherTextStealingMultipart`, here we only wanted to add some cases in line with the CTR addition (4ce804890912ce7a0002c9e631c4dc699ac33c39).

Reply:
Ok.